test/operations/job.go

```diff
@@ -30,15 +30,6 @@ import (
 
 type OpsJobStatus string
 
-const (
```

Comment:
Do we need this file at all anymore?

Reply:
Yes, this is the fix that @capri-xiyue did to e2e tests. I accidentally removed it, but this one in particular is being used for e2e tests. It's bad, I know :) I moved it inside `test/ `, to avoid confusions, and not used it anywhere else.
@chizhg and Peter will take care of refactoring all this stuff!